component/filter/filter.css

```diff
@@ -0,0 +1,83 @@
+.filter__left{
```

Comment:
use space before { its good for indentation,use beautify extension for this in vs code